packages/harpguru-core/src/hooks/use-interpolate-option-stack-transition-value/use-interpolate-option-stack-transition-value.ts

```diff
@@ -1,17 +1,18 @@
 import { interpolate } from 'react-native-reanimated'
-import type { Node } from 'react-native-reanimated'
 
 import { getInputRange, getOutputRange } from '../../utils'
 
+// TODO: This is not a hook. It should be renamed.
+// I feel like this might not even be necessary in some
+// cases. The OptionList component for example does
+// another interpolation, so what's the value.
```

Comment:
The goal isn't to make too many changes to the way things work. We're just trying to get an upgrade done. This is tech debt. If there isn't too much tech debt at the end of this merge request I might fix it. Otherwise I'll gather it all in to a separate enhancement.

Reply:
I'll leave this TODO in as an in place reminder but I'll remove the other two as they are a bit less important and more speculative.

Reply:
Issue raised in https://github.com/js-jslog/harpguru-lift-shift-sdk-47/issues/9